python/orca/example/learn/pytorch/super_resolution/super_resolution.py

```diff
@@ -57,6 +57,7 @@
 parser.add_argument('--backend', type=str, default="bigdl",
                     help='The backend of PyTorch Estimator; '
                          'bigdl and torch_distributed are supported.')
+parser.add_argument('--data_dir', type=str,required=True, help='The path of datesets.')                         
```

Comment:
Specifiy a default path parser.add_argument('--data_dir', type=str, default="./dataset", help='The path of datesets.') better?
or update jenkins test script by adding `--data_dir` in https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/orca/dev/example/run-example-test-ray.sh#L80